client/landing/gutenboarding/hooks/use-on-login.ts

```diff
@@ -4,6 +4,7 @@
 import * as React from 'react';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useI18n } from '@automattic/react-i18n';
+import { isEnabled } from 'config';
```

Comment:
I've put the `config` imports under `External dependencies`, since they're arguably external to Gutenboarding.

Reply:
I appreciate you'd _want_ it to be external (and I agree it could be), but both Gutenboarding and Config are under `client`, really not making it that external after all? :-)

Reply:
Yeah, fair point. I'll move to Internal (after going with [this change](https://github.com/Automattic/wp-calypso/pull/43246/files#r440779093)). I _think_ however that I've already seen a few examples of this elsewhere in Gutenboarding :grimacing: